priv/templates/phoenix.gen.json/changeset_view.ex

```diff
@@ -1,9 +1,18 @@
 defmodule <%= base %>.ChangesetView do
   use <%= base %>.Web, :view
 
+  @doc """
+  Traverses and translates changeset errors.
+
+  See `Ecto.Changeset.traverse_errors/2` for more details.
```

Comment:
Should we say:

    See `Ecto.Changeset.traverse_errors/2` and
    `MyApp.ErrorHelpers.translate_error/1` for more details.

?